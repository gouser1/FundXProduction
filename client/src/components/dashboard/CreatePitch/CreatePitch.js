import React, { useState, useEffect } from "react";
import { Grid, Container, Hidden, Typography } from "@material-ui/core";
import { Formik, Form } from "formik";
import * as Yup from "yup";
import axios from "axios";
import { useHistory, Link } from "react-router-dom";
import Textfield from "../../FormsUI/Textfield/index";
import Tooltip from "@material-ui/core/Tooltip";
import Select from "../../FormsUI/Select/index";
import Button from "../../FormsUI/Button/index";
import countries from "../../../data/countries.json";
import locations from "../../../data/locations.json";
import industries from "../../../data/industries.json";
import investmentRoles from "../../../data/investmentRoles.json";
import capitalAmounts from "../../../data/capitalAmounts.json";
import { AuthContext } from "../../../helpers/AuthContext";
import useStyles from "./CreatePitchStyle";

function CreatePitch(props) {
  let history = useHistory();
  const classes = useStyles();

  const INITIAL_FORM_STATE = {
    pitchTitle: "",
    contactEmail: "",
    website: "",
    location: "",
    country: "",
    industry: "",
    industry2: "",
    idealInvestmentRole: "",
    capitalNeeded: "",
    capitalRaised: "",
    pitchInfo: "",
  };

  const [authState, setAuthState] = useState({
    displayName: "",
    id: 0,
    status: false,
  });

  useEffect(() => {
    if (!localStorage.getItem("accessToken")) {
      history.push("/login");
    }
  }, [history]);

  const FORM_VALIDATION = Yup.object().shape({
    pitchTitle: Yup.string()
      .required("Please enter a pitch title")
      .max(25)
      .min(10),
    website: Yup.string().max(100),
    contactEmail: Yup.string().email().required("Required").max(50),
    location: Yup.string().required("Please select your location"),
    country: Yup.string().required("Please select your country"),
    industry: Yup.string().required("Please select an industry"),
    industry2: Yup.string(),
    idealInvestmentRole: Yup.string().required(
      "Please select an investment role"
    ),
    capitalNeeded: Yup.string().required("Please select capital needed"),
    capitalRaised: Yup.string().required("Please select capital raised"),
    pitchInfo: Yup.string()
      .required("Please enter your pitch info")
      .max(5000)
      .min(100),
  });

  return (
    <AuthContext.Provider value={{ authState, setAuthState }}>
      <Grid container>
        <Grid item xs={10} lg={6} style={{ paddingTop: "2%" }}>
          <Container maxWidth="md">
            <div className={classes.formWrapper}>
              <Formik
                initialValues={INITIAL_FORM_STATE}
                validationSchema={FORM_VALIDATION}
                onSubmit={(data) => {
                  axios
                    .post("http://localhost:3001/pitches", data, {
                      headers: {
                        accessToken: localStorage.getItem("accessToken"),
                      },
                    })
                    .then((response) => {
                      history.push("/dashboard/mypitches");
                    });
                }}
              >
                <Form>
                  <Grid container spacing={2}>
                    <Grid item xs={12}>
                      <Typography
                        className={classes.h1}
                        style={{ marginBottom: "10%" }}
                      >
                        {" "}
                        Create Pitch
                      </Typography>
                      <Typography className={classes.h2}>
                        {" "}
                        Pitch Title
                      </Typography>
                      <Tooltip
                        title="What are you going to call your pitch? "
                        placement="top-end"
                      >
                        <Textfield name="pitchTitle"></Textfield>
                      </Tooltip>

                      <Typography className={classes.h2}>
                        {" "}
                        Website (Optional)
                      </Typography>
                      <Tooltip
                        title="Please include the full web address. "
                        placement="top-end"
                      >
                        <Textfield name="website"></Textfield>
                      </Tooltip>

                      <Typography className={classes.h2}>
                        {" "}
                        Contact Email
                      </Typography>
                      <Tooltip
                        title="This email is how investors will contact you"
                        placement="top-end"
                      >
                        <Textfield name="contactEmail"></Textfield>
                      </Tooltip>

                      <Typography className={classes.h2}> Location</Typography>
                      <Select name="location" options={locations}></Select>

                      <Typography className={classes.h2}> Country</Typography>
                      <Select name="country" options={countries}></Select>

                      <Typography className={classes.h2}> Industry</Typography>
                      <Tooltip
                        title="Which industry best suits your business? "
                        placement="top-end"
                      >
                        <Select name="industry" options={industries}></Select>
                      </Tooltip>

                      <Typography className={classes.h2}>
                        {" "}
                        Industry 2 (optional)
                      </Typography>
                      <Tooltip
                        title="A 2nd option will define your pitch for more investors "
                        placement="top-end"
                      >
                        <Select name="industry2" options={industries}></Select>
                      </Tooltip>

                      <Typography className={classes.h2}>
                        {" "}
                        Ideal Investment Role
                      </Typography>
                      <Tooltip
                        title="How active woul you like your investor to be?"
                        placement="top-end"
                      >
                        <Select
                          name="idealInvestmentRole"
                          options={investmentRoles}
                        ></Select>
                      </Tooltip>
                      <Typography className={classes.h2}>
                        {" "}
                        Capital needed
                      </Typography>
                      <Select
                        label="£"
                        name="capitalNeeded"
                        options={capitalAmounts}
                      ></Select>

                      <Typography className={classes.h2}>
                        {" "}
                        Capital raised
                      </Typography>
                      <Select
                        name="capitalRaised"
                        label="£"
                        options={capitalAmounts}
                      ></Select>

                      <Typography className={classes.h2}>
                        {" "}
                        Pitch Info
                      </Typography>
                      <Tooltip
                        title="This section is where you sell your pitch so don't be shy! "
                        placement="top-end"
                      >
                        <Textfield
                          multiline
                          rows={6}
                          name="pitchInfo"
                          style={{ paddingBottom: "2%" }}
                        ></Textfield>
                      </Tooltip>
                      <Button>
                        {" "}
                        <Typography
                          className={classes.p2}
                          style={{ color: "white" }}
                        >
                          Post Pitch
                        </Typography>
                      </Button>
                    </Grid>
                  </Grid>
                </Form>
              </Formik>
            </div>
          </Container>
        </Grid>
        <Hidden mdDown>
          <Grid item xs={10} lg={6}>
            <Container
              maxWidth="md"
              style={{ backgroundColor: "#edf0ef", height: "100%" }}
            >
              <Container style={{ paddingTop: "6%" }}>
                <Typography
                  className={classes.p1}
                  style={{
                    backgroundColor: "#d4e3dd",
                    padding: "4px",
                    fontWeight: "500",
                  }}
                >
                  Do you have any questions? Click <Link to="/">here</Link> to
                  view some helpful FAQ answers.
                </Typography>
              </Container>
            </Container>
          </Grid>
        </Hidden>
      </Grid>
    </AuthContext.Provider>
  );
}

export default CreatePitch;
