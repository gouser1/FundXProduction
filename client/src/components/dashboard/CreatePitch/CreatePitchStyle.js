import { makeStyles } from "@material-ui/styles";

export default makeStyles(() => ({
  root: {
    flexGrow: 1,
  },
  typography: {
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.2em",
    color: "#46637f",
  },
  button: {
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.2em",
    color: "white",
    backgroundColor: "#3bc693",
    "&:hover": {
      backgroundColor: "#34b384",
    },
    textTransform: "none",
  },
  h1: {
    textAlign: "center",
    fontFamily: "'Karla', sans-serif;",
    fontSize: "1.5em",
    color: "#22303d",
    fontWeight: "700",
    "@media (min-width:700px)": {
      fontSize: "2rem",
    },
  },
  h2: {
    fontFamily: "'Rubik', sans-serif;",
    fontSize: "0.9em",
    color: "black",
    fontWeight: "700",
    marginTop: "2%",
  },
}));
