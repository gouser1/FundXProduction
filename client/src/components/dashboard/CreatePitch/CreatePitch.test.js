import React from "react";
import renderer from "react-test-renderer";

import CreatePitch from "./CreatePitch";

it("renders correctly", () => {
  const tree = renderer.create(<CreatePitch />).toJSON();
  expect(tree).toMatchSnapshot();
});
