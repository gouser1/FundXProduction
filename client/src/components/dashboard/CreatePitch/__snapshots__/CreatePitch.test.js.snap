// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-lg-6"
    style={
      Object {
        "paddingTop": "2%",
      }
    }
  >
    <div
      className="MuiContainer-root MuiContainer-maxWidthMd"
    >
      <div>
        <form
          action="#"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
          >
            <div
              className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
            >
              <p
                className="MuiTypography-root makeStyles-h1-4 MuiTypography-body1"
                style={
                  Object {
                    "marginBottom": "10%",
                  }
                }
              >
                 
                Create Pitch
              </p>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 
                Pitch Title
              </p>
              <div
                aria-describedby={null}
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                onMouseLeave={[Function]}
                onMouseOver={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                title="What are you going to call your pitch? "
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <input
                    aria-invalid={false}
                    autoFocus={false}
                    className="MuiInputBase-input MuiOutlinedInput-input"
                    disabled={false}
                    name="pitchTitle"
                    onAnimationStart={[Function]}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    required={false}
                    type="text"
                    value=""
                  />
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 
                Website (Optional)
              </p>
              <div
                aria-describedby={null}
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                onMouseLeave={[Function]}
                onMouseOver={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                title="Please include the full web address. "
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <input
                    aria-invalid={false}
                    autoFocus={false}
                    className="MuiInputBase-input MuiOutlinedInput-input"
                    disabled={false}
                    name="website"
                    onAnimationStart={[Function]}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    required={false}
                    type="text"
                    value=""
                  />
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 
                Contact Email
              </p>
              <div
                aria-describedby={null}
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                onMouseLeave={[Function]}
                onMouseOver={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                title="This email is how investors will contact you"
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <input
                    aria-invalid={false}
                    autoFocus={false}
                    className="MuiInputBase-input MuiOutlinedInput-input"
                    disabled={false}
                    name="contactEmail"
                    onAnimationStart={[Function]}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    required={false}
                    type="text"
                    value=""
                  />
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 Location
              </p>
              <div
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="mui-component-select-location"
                    className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="mui-component-select-location"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </div>
                  <input
                    aria-hidden={true}
                    autoFocus={false}
                    className="MuiSelect-nativeInput"
                    name="location"
                    onAnimationStart={[Function]}
                    onChange={[Function]}
                    required={false}
                    tabIndex={-1}
                    value=""
                  />
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 Country
              </p>
              <div
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="mui-component-select-country"
                    className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="mui-component-select-country"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </div>
                  <input
                    aria-hidden={true}
                    autoFocus={false}
                    className="MuiSelect-nativeInput"
                    name="country"
                    onAnimationStart={[Function]}
                    onChange={[Function]}
                    required={false}
                    tabIndex={-1}
                    value=""
                  />
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 Industry
              </p>
              <div
                aria-describedby={null}
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                onMouseLeave={[Function]}
                onMouseOver={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                title="Which industry best suits your business? "
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="mui-component-select-industry"
                    className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="mui-component-select-industry"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </div>
                  <input
                    aria-hidden={true}
                    autoFocus={false}
                    className="MuiSelect-nativeInput"
                    name="industry"
                    onAnimationStart={[Function]}
                    onChange={[Function]}
                    required={false}
                    tabIndex={-1}
                    value=""
                  />
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 
                Industry 2 (optional)
              </p>
              <div
                aria-describedby={null}
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                onMouseLeave={[Function]}
                onMouseOver={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                title="A 2nd option will define your pitch for more investors "
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="mui-component-select-industry2"
                    className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="mui-component-select-industry2"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </div>
                  <input
                    aria-hidden={true}
                    autoFocus={false}
                    className="MuiSelect-nativeInput"
                    name="industry2"
                    onAnimationStart={[Function]}
                    onChange={[Function]}
                    required={false}
                    tabIndex={-1}
                    value=""
                  />
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 
                Ideal Investment Role
              </p>
              <div
                aria-describedby={null}
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                onMouseLeave={[Function]}
                onMouseOver={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                title="How active woul you like your investor to be?"
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="mui-component-select-idealInvestmentRole"
                    className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="mui-component-select-idealInvestmentRole"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </div>
                  <input
                    aria-hidden={true}
                    autoFocus={false}
                    className="MuiSelect-nativeInput"
                    name="idealInvestmentRole"
                    onAnimationStart={[Function]}
                    onChange={[Function]}
                    required={false}
                    tabIndex={-1}
                    value=""
                  />
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 
                Capital needed
              </p>
              <div
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
              >
                <label
                  className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                  data-shrink={false}
                >
                  £
                </label>
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="mui-component-select-capitalNeeded"
                    className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="mui-component-select-capitalNeeded"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </div>
                  <input
                    aria-hidden={true}
                    autoFocus={false}
                    className="MuiSelect-nativeInput"
                    name="capitalNeeded"
                    onAnimationStart={[Function]}
                    onChange={[Function]}
                    required={false}
                    tabIndex={-1}
                    value=""
                  />
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      className="PrivateNotchedOutline-legendLabelled-8"
                    >
                      <span>
                        £
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 
                Capital raised
              </p>
              <div
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
              >
                <label
                  className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
                  data-shrink={false}
                >
                  £
                </label>
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl"
                  onClick={[Function]}
                >
                  <div
                    aria-haspopup="listbox"
                    aria-labelledby="mui-component-select-capitalRaised"
                    className="MuiSelect-root MuiSelect-select MuiSelect-selectMenu MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
                    id="mui-component-select-capitalRaised"
                    onBlur={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    role="button"
                    tabIndex={0}
                  >
                    <span
                      dangerouslySetInnerHTML={
                        Object {
                          "__html": "&#8203;",
                        }
                      }
                    />
                  </div>
                  <input
                    aria-hidden={true}
                    autoFocus={false}
                    className="MuiSelect-nativeInput"
                    name="capitalRaised"
                    onAnimationStart={[Function]}
                    onChange={[Function]}
                    required={false}
                    tabIndex={-1}
                    value=""
                  />
                  <svg
                    aria-hidden={true}
                    className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7 10l5 5 5-5z"
                    />
                  </svg>
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      className="PrivateNotchedOutline-legendLabelled-8"
                    >
                      <span>
                        £
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <p
                className="MuiTypography-root makeStyles-h2-5 MuiTypography-body1"
              >
                 
                Pitch Info
              </p>
              <div
                aria-describedby={null}
                className="MuiFormControl-root MuiTextField-root MuiFormControl-fullWidth"
                onMouseLeave={[Function]}
                onMouseOver={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "paddingBottom": "2%",
                  }
                }
                title="This section is where you sell your pitch so don't be shy! "
              >
                <div
                  className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline"
                  onClick={[Function]}
                >
                  <textarea
                    aria-invalid={false}
                    autoFocus={false}
                    className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline"
                    disabled={false}
                    name="pitchInfo"
                    onAnimationStart={[Function]}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    required={false}
                    rows={6}
                    value=""
                  />
                  <fieldset
                    aria-hidden={true}
                    className="PrivateNotchedOutline-root-6 MuiOutlinedInput-notchedOutline"
                    style={
                      Object {
                        "paddingLeft": 8,
                      }
                    }
                  >
                    <legend
                      className="PrivateNotchedOutline-legend-7"
                      style={
                        Object {
                          "width": 0.01,
                        }
                      }
                    >
                      <span
                        dangerouslySetInnerHTML={
                          Object {
                            "__html": "&#8203;",
                          }
                        }
                      />
                    </legend>
                  </fieldset>
                </div>
              </div>
              <button
                className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
                disabled={false}
                fullwidth={true}
                onBlur={[Function]}
                onClick={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                type="button"
              >
                <span
                  className="MuiButton-label"
                >
                   
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                    style={
                      Object {
                        "color": "white",
                      }
                    }
                  >
                    Post Pitch
                  </p>
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;
